azext_iot/digitaltwins/_help.py

```diff
@@ -32,33 +32,33 @@ def load_digitaltwins_help():
 
```

Comment:
We need a HISTORY.rst entry.

Reply:
will be in the follow up pr

Reply:
It was added as part of the 0.18.2 release.